spec/abstract/ARM/Arch_Structs_A.thy

```diff
@@ -330,6 +330,9 @@ definition parse_time_arg :: "nat \<Rightarrow> data list \<Rightarrow> time" wh
 definition words_from_time :: "time \<Rightarrow> machine_word list" where
   "words_from_time t = [ucast t, ucast (t >> 32)]"
 
+definition TIME_ARG_SIZE :: nat where
+  "TIME_ARG_SIZE \<equiv> 2" (* sizeof(ticks_t) / sizeof(word_t) *)
```

Comment:
That comment is exciting, because it would actually make that definition generic. If you define it as what the comment says, only the C version of this will need to be arch specific. Not sure it does that much since `words_from_time` still will need to be arch specific, but you could add a comment with `FIXME arch-split` that this could be a generic definition so we remember when we get there.